src/index.js

```diff
@@ -369,14 +430,100 @@ export const EDSCTimeline = ({
     }
   }
 
+  /**
+   * Generates new timeIntervals at the given zoom and moves the timeline to the given timestamp
+   * @param {Integer} timestamp Timestamp to zoom to
+   * @param {Integer} zoom Zoom level to zoom to
+   * @param {Integer} offset Optional: offset the timeline position by this offset value
+   */
+  const zoomToTimestamp = (timestamp, zoom, offset = 0) => {
+    const newIntervals = [
```

Comment:
May want to separate out the values of `newIntervals` into named variables for clarity and then create the array with the named variables.